app/api/packets/helpers.ts

```diff
@@ -6,9 +6,8 @@ import { CachingJsonRpcProvider } from '@/api/utils/provider';
 import Abi from '@/utils/dispatcher.json';
 import { GetTmClient } from '@/api/utils/cosmos';
 import { Packet, PacketStates } from '@/utils/types/packet';
-import { NextResponse } from 'next/server';
 
-export async function getPackets() {
```

Comment:
I think it's okay to fully remove it with the cache api as well

Reply:
Get rid of all the previous packets functionality, including the cache?

Reply:
Yes